src/yuzu/configuration/config.cpp

```diff
@@ -293,6 +300,26 @@ void Config::ReadPlayerValues() {
                 player_analogs = default_param;
             }
         }
+
+        for (int i = 0; i < player.motion_devices.size(); ++i) {
```

Comment:
```suggestion
        for (std::size_t i = 0; i < player.motion_devices.size(); ++i) {
```

Otherwise this will cause sign conversion warnings.